app/selectors/autocomplete.js

```diff
@@ -224,3 +224,21 @@ export const filterPrivateChannels = createSelector(
         return channels.map((c) => c.id);
     }
 );
+
+export const getMatchTermForDateMention = (() => {
```

Comment:
FYI, there's a `memoizeResult` helper function in `mattermost-redux/src/utils/helpers` if you need to do this in the future. This is fine though since it's just copied from above that was also written before `memoizeResult` existed

Reply:
cool, thanks, yeah, I'm just following the existing pattern